DesktopEdge/Views/Screens/MFAScreen.xaml

```diff
@@ -48,7 +48,7 @@
                 <RowDefinition Height="10"></RowDefinition>
                 <RowDefinition Height="30"></RowDefinition>
             </Grid.RowDefinitions>
-            <Label Content="Setup MFA" Grid.Row="0" FontSize="16" HorizontalAlignment="Center" FontWeight="Bold"></Label>
+            <Label Content="SETUP MFA" Foreground="white" Grid.Row="0" FontSize="14" HorizontalAlignment="Center" FontFamily="/ZitiDesktopEdge;component/Assets/Fonts/#Russo One"></Label>
```

Comment:
Better alignment with the visualization across other screen.  Change of font and sizing to be more consistent with other screens.  Minor updates and likely could use another pass for full alignment.
![image](https://github.com/user-attachments/assets/f2ddd89a-54a1-487a-b69f-0f00838cf616)